codexctl/sync.py

```
@@ -1,243 +1,243 @@
+
+        if self.logger is None:
+            self.logger = logging
+
+        self.BASE = BASE
+        self.ID_ATTRIBUTE = "ID"
+        self.NAME_ATTRIBUTE = "VissibleName"
+        self.MTIME_ATTRIBUTE = "ModifiedClient"
+
+        self.logger.debug(f"Base is: {BASE}")
+
+    def __POST(self, endpoint, data={}, fileUpload=False):
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)